src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -326,13 +327,17 @@ class WelcomePage extends Disposable {
 			workspaces = workspaces.filter(recent => !this.contextService.isCurrentWorkspace(isRecentWorkspace(recent) ? recent.workspace : recent.folderUri));
 			if (!workspaces.length) {
 				const recent = container.querySelector('.welcomePage') as HTMLElement;
+				const moreRecent = document.querySelector('.moreRecent') as HTMLElement;
```

Comment:
This ideally shouldn't be at the document level - can't you scope it down further?